_data/harnesses/finding-distinct/kstreams.yml

```diff
@@ -0,0 +1,162 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - change_directory: merge-streams
```

Comment:
Dir should be `distinct-events`, to match `init.sh`

Reply:
Thanks for the review @colinhicks ...

The `init.sh` script has a `cd distinct-events` inside of it.  Do I still want this step to do the `change_directory`?   I think I need to figure out how to run the test harness so I can verify all this myself.  Any tips for that?

Reply:
@rspurgeon `make -C _includes/tutorials/finding-distinct/kstreams/code tutorial`

I will track down why we are invoking both the `cd distinct-events` and the `change_directory` step elsewhere.